### PR TITLE
Dynamic change dashboard severity label

### PR DESCRIPTION
**Pull request by Mnakagawa**
Related: #2242

### CODE REVIEW

**Comment by Mnakagawa** [comment]
I will fix some tests after confirm the travis test.

File: client/static/js/dashboard_view.js [hunk 1]
```diff
@@ -285,12 +316,23 @@ var DashboardView = function(userProfile) {
   }
 
   function load() {
+    self.startConnection('severity-rank', loadSeverityLabel);
```
Comment (cosmo0920) [inline client/static/js/dashboard_view.js]: Each of tables are independent?
If so, current implementation is enough.

Comment (Mnakagawa) [inline client/static/js/dashboard_view.js]: Yes. dashboard page has some tables.
The tables are independent.

Comment (cosmo0920) [inline client/static/js/dashboard_view.js]: :+1: 
Awesome!

**Comment by cosmo0920** [comment]
Looks good. Can we merge this?

**Comment by Mnakagawa** [comment]
OK（maybe...）

**Comment by cosmo0920** [comment]
> OK（maybe...）

😖

**Comment by cosmo0920** [comment]
Merged!

**Pull request merged**